posthog/hogql_queries/utils/recordings_helper.py

```diff
@@ -0,0 +1,112 @@
+from collections import defaultdict
+from datetime import datetime
+from collections.abc import Iterable
+
+from posthog.hogql import ast
+from posthog.hogql.query import execute_hogql_query
+from posthog.models import Team
+from posthog.session_recordings.models.session_recording import SessionRecording
+from posthog.session_recordings.queries.session_replay_events import ttl_days
+
+
+class RecordingsHelper:
```

Comment:
not a massive fan of "helper" in general but at least now the file isn't only called `recordings` which is a little too general :)